attributes/sdk.rb

```diff
@@ -91,6 +91,8 @@
 default['cdap']['sdk']['install_path'] = '/opt/cdap'
 default['cdap']['sdk']['user'] = 'cdap'
 default['cdap']['sdk']['manage_user'] = true
+default['cdap']['sdk']['profile_d']['node_env'] = 'production'
```

Comment:
I think the default should empty. If I'm not mistaken, production is for distributed and it's unset in sdk.  (Should also confirm "" treated the same as unset)